src/content/docs/more-integrations/open-source-telemetry-integrations/opentelemetry/opentelemetry-comparison.mdx

```diff
@@ -0,0 +1,1139 @@
+
+OpenTelemetry services use spans to create golden signals, whereas New Relic agent services use metrics.
+
+<CollapserGroup>
+  <Collapser
+    className="freq-link"
+    id="web-trx"
+    title="Web transactions time breakdown"
+  >
+    New Relic agents: ✅ &nbsp; &nbsp; &nbsp; OpenTelemetry: ❌
+
+    OpenTelemetry does not specify or collect metrics breaking down service response time.
+  </Collapser>
+</CollapserGroup>
+
+## Filter/group data in the UI [#filter]
+
+New Relic agents:  🟡 (Limited) &nbsp; &nbsp; &nbsp; OpenTelemetry: ✅
+
+OpenTelemetry’s service UIs in New Relic let you filter/group data in charts using any attribute. In New Relic One, only OpenTelemetry services have the filter bars shown below:
+
+<img
+  title="Screenshot showing the filter bars for OTel"
+  alt="Screenshot showing the filter bars for OTel"
+  src={otelFilterBars}
+/>
+
+In contrast, the New Relic agent-based UI can be filtered/grouped by a more limited set of attributes. For example, all data can be filtered by host/instance. Several specialized views provide fixed groupings of their metrics, for example: by database query, by transaction/endpoint, by external service call, by error message and exception type, etc. In addition, event data available for many of these views can be filtered/grouped by any attribute.
+
+## Distributed tracing [#dt]
+
+New Relic agents: ✅ &nbsp; &nbsp; &nbsp; OpenTelemetry: ✅
+
+Distributed tracing is supported today by our agents and OpenTelemetry. Both provide W3C Trace Context compatibility. Furthermore, OpenTelemetry supports W3C Baggage, highly configurable sampling options, and events on spans, or `span events` as they're referred to in New Relic.
+
+<CollapserGroup>
+  <Collapser
+    className="freq-link"
+    id="trace-context"
+    title="W3C Trace Context"
+  >
+    New Relic agents: ✅ &nbsp; &nbsp; &nbsp; OpenTelemetry: ✅
+
+    This support means that in a distributed system where some services are instrumented with OpenTelemetry and others with APM agents, trace context can be propagated successfully from one service to the next. This gives you visibility into activity across the system even when you use more than one instrumentation technology.
+  </Collapser>
+  <Collapser
+    className="freq-link"
+    id="baggage"
+    title="W3C Baggage"
+  >
+    New Relic agents:  ❌ &nbsp; &nbsp; &nbsp; OpenTelemetry: ✅
+
+    In OpenTelemetry, baggage is a mechanism to propagate `key:value` pairs. It allows for observability events in a downstream service to be indexed with attributes provided by an upstream service in the same transaction/trace, without having to pull the attributes from a database. For instance, a user ID added as baggage will be accessible to downstream services, and they can then also index their spans and metrics by the user ID.
```

Comment:
CNCF OTel docs do use "downstream", but we have noted some conflict about these terms in user understanding. Perhaps "...events in a downstream service to be indexed with attributes provided by [a service making the request] in the same..." and "...ID added as baggage will be accessible to [services where requests are sent], and..."